src/connection/database/actions.js

```diff
@@ -175,6 +182,29 @@ function autoLogInError(id, error) {
   });
 }
 
+function successCaptchaRequired(id) {
+  swap(updateEntity, 'lock', id, m => {
+    if (!hasScreen(m, 'login')) {
+      return l.setSubmitting(m, false);
+    }
+
+    // _ prevents cleaning the fields
+    m = l.setSubmitting(setScreen(m, 'login', ['_']), false);
+
+    let message = React.createElement(
+      'div',
+      null,
+      i18n.html(m, ['success', 'signUp']),
+      React.createElement('br'),
+      l.captcha(m).get('type') === 'code'
```

Comment:
“Enter the code shown above” that string is fine for the hint of the captcha input. But when used on the green banner on the top, there’s no code "above". It should be _below_. 

Alternatively, instead of reusing the existing text, add a new entry for “Complete the captcha” (or something similar)

Reply:
oh you are right!